webpack/module-minifier-plugin/package.json

```diff
@@ -17,7 +17,9 @@
   },
   "peerDependencies": {
     "webpack": "^4.31.0",
-    "webpack-sources": "~1.4.3",
+    "webpack-sources": "~1.4.3"
+  },
+  "optionalPeerDependencies": {
```

Comment:
Looks like this is the correct way to specify an optional peerDependency: https://github.com/microsoft/rushstack/blob/master/webpack/localization-plugin/package.json#L20-L27